src/collection.jl

```diff
@@ -253,6 +221,8 @@ function normalform(coll::Collector{T}, v::SyllableVector{T}) where T <: OscarIn
         end
         if b > 1
           push!(w, (y, b-1))
+        else
+          push!(w, (y, b+1))
```

Comment:
only if `b < -1` holds

Reply:
no, we don't. we call `freely_reduced` enough times and that removes entries with zero exponent

Reply:
O.k., let us merge the fix.